tmva/pymva/src/RModelParser_Keras.cxx

```diff
@@ -481,14 +519,16 @@ RModel Parse(std::string filename){
       // inference code.
       else if(fLayerType == "Dense")
          rmodel.AddBlasRoutines({"Gemm", "Gemv"});
+      else if (fLayerType == "BatchNormalization") 
```

Comment:
Thank you for adding this! It is needed for the BatchNorm. 
Strange it was working before without these declarations.

Reply:
I think this was already there in the ONNX Parser, but for supporting this in Keras, I had to add it this way.